app/src/main/res/values/ids.xml

```diff
@@ -47,4 +47,5 @@
     <item name="action_set_as_wallpaper" type="id"/>
     <item name="action_remove_file" type="id"/>
     <item name="action_pin_to_homescreen" type="id"/>
+    <item name="action_rotate_image" type="id"/>
```

Comment:
Required to avoid code issue during merge process.
Will be vanished when Rotate PR is merged.